.github/travis/common.sh

```diff
@@ -51,6 +51,32 @@ if [ -z "$DATESTR" ]; then
 	fi
 fi
 
+function make_target() {
+  target=$1
+
+  if [ ! -v MAKE_JOBS ]; then
+    export MAKE_JOBS=$(nproc)
+    echo "Setting MAKE_JOBS to $MAKE_JOBS"
+  fi;
+
+  export VPR_NUM_WORKERS=${MAKE_JOBS}
+
+  start_section "symbiflow.$target" "$2"
+  make_status=0
```

Comment:
Any reason this is lowercase?

Reply:
local to the function, but no strong feeling on it.